### PR TITLE
finish home work 3

### PR DESCRIPTION
**Pull request by nikita12012005**
finish

### CODE REVIEW

File: hw3.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# 1_exercise
+odd = [1, 3, 5, 7]
+even = [2, 4, 6, 8]
+
+for index, value in zip(odd, even):
+    print(f'index: {index}, value: {value}')
```
Comment (bohdandovbsyh) [inline hw3.py:6]: please read the task again. IYou did not do what is explained there

File: hw3.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+print(division_way_two)
+
+#3_exercise
+friends = ["John", "Marta", "James"]
+enemies = ["John", "Johnatan", "Artur"]
+for friend, enemie in zip(friends, enemies):
+    if friend== enemie:
+        print(f'{friend} we are not friends anymore')
+    elif friend!=enemie and friend!='James':
+        print(f'{friend} we are the best friends')
```
Comment (bohdandovbsyh) [inline hw3.py:25]: add empty line at the end of file

**Review by nikita12012005 — COMMENTED** [review]
finish nove exercise

File: hw3.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+print(multiplication_way_two)
+division_way_one= 2/2
+print(division_way_one)
+division_way_two = 2//2
+print(division_way_two)
+
+#3_exercise
+friends = ["John", "Marta", "James"]
+enemies = ["John", "Johnatan", "Artur"]
+for friend, enemie in zip(friends, enemies):
```
Comment (bohdandovbsyh) [inline hw3.py:33]: not a good idea. what if "John" will be in a random place?

**Pull request merged**